app/views/bill-runs/setup/region.njk

```diff
@@ -27,35 +26,33 @@
       }) }}
   {% endif %}
 
-  <div class="govuk-body">
-    <form method="post">
-      <input type="hidden" name="wrlsCrumb" value="{{wrlsCrumb}}"/>
+  <form method="post">
+    <input type="hidden" name="wrlsCrumb" value="{{wrlsCrumb}}"/>
 
-      {% set regionItems = [] %}
-      {% for region in regions %}
-        {% set regionItem = { text: region.displayName, value: region.id, checked: region.id === selectedRegion } %}
+    {% set regionItems = [] %}
+    {% for region in regions %}
+      {% set regionItem = { text: region.displayName, value: region.id, checked: region.id === selectedRegion } %}
 
-        {# Push our item into the region items array #}
-        {% set regionItems = (regionItems.push(regionItem), regionItems) %}
-      {% endfor %}
+      {# Push our item into the region items array #}
```

Comment:
do we need this comment? I believe the code is descriptive enough and this doesn't add anything